crates/pera-types/src/lib.rs

```diff
@@ -88,6 +88,7 @@ pub mod versioned;
 pub mod zk_login_authenticator;
 pub mod zk_login_util;
 
+pub mod messages_signature_mpc;
```

Comment:
1. missing empty line here
2. I think the above list is in alphabetical order. Let's put it in there, shall we?

Reply:
fixed